client/extensions/zoninator/components/forms/zone-details-form/index.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { reduxForm } from 'redux-form';
+import { translate } from 'i18n-calypso';
```

Comment:
I believe this is considered legacy and the preferred method is to use the `localize` HOC.